### DISCUSSION TITLE
Wondering if this will work on a Raspberry Pi 2 (2015) model

### DISCUSSION THREAD

**Discussion by ando01** [MLB-LED-Scoreboard/mlb-led-scoreboard#541] [Q&A] [answered]
I saw in the README that the Zero is not recommended due to slowness issues. I am wondering if this is related to the CPU or memory as the Zero only has 512MB RAM. I have a couple of old Raspberry Pi 2 B's from 2015 that have a 900MHz CPU with 1GB of RAM. Has anyone tried this project on them or know if it is more RAM or CPU hungry?

Thanks,
Andy

**Comment by c2mfj** [comment, marked as answer]
I’ve put it on a 2b before with no issues.

**Reply by ando01** [reply]
Awesome! Thanks for letting me know.

**Comment by ty-porter** [comment]
The installer used to hang on the Zero models (https://github.com/MLB-LED-Scoreboard/mlb-led-scoreboard/issues/283) possibly due to running out of memory. However, I _think_ we fixed that by installing via `apt`.

Regardless, a 2B should be able to run it.
